.github/workflows/browserslist-db-update.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           git config --global user.name "GitHub Actions"
 
       - name: Update Browserslist database and create PR if applicable
-        uses: c2corg/browserslist-update-action@v1
+        uses: c2corg/browserslist-update-action@v2
```

Comment:
That is correct, thank you for spotting, @aaemnnosttv.

@jimmymadon Unfortunately, the `v2` of `c2corg/browserslist-update-action` wouldn't work in our case because it requires a higher Node & NPM version to function. Until that is updated, we will need to stick with `v1` for now.

More context [here](https://github.com/google/site-kit-wp/issues/5345#issuecomment-1256874079).

Reply:
That is correct, thank you for spotting, @aaemnnosttv.

@jimmymadon Unfortunately, the `v2` of `c2corg/browserslist-update-action` wouldn't work in our case because it requires a higher Node & NPM version to function. Until that is updated, we will need to stick with `v1` for now.

More context [here](https://github.com/google/site-kit-wp/issues/5345#issuecomment-1256874079).